src/main/java/com/api/Timesheets/repositories/UserRepo.java

```diff
@@ -12,4 +12,6 @@
   User findByUsernameIgnoreCase(String userName);
   Optional<User> findByEmail(String email);
 
+  User findByRole(String role);
```

Comment:
maybe we can put returnType as Optional<User> so that we can make use of optinal?